features/edit_sections.feature

```diff
@@ -12,8 +12,6 @@ Feature: Edit Sections
     And I should see "Healthcare Concept"
     When I click on the "Edit" link
     And I fill in the "section-name" field with "Edited Section"
-    And I fill in the "Tags" field with "TagTest1"
-    And I tab out of the "Tags" field
```

Comment:
This PR looks good I am just wondering why these tests were removed before we merge, was it potentially catching a bug?

Reply:
Build was failing in Jenkins on info-modal:
Failing Scenarios:
cucumber features/edit_sections.feature:4 # Scenario: Edit a Draft Section
cucumber features/manage_surveys.feature:55 # Scenario: Revise Survey

When running the scenario locally, Error was noting the Tag button was not enabled.  For the sake of the sprint and getting the info-modal and CAG metric in, removed and planned to discuss and debug further.

Just re-ran with the tag code above on local to reproduce the detailed error.  Scenario passed?

Can discuss further.